src/lockscreen.js

```diff
@@ -0,0 +1,81 @@
+/* exported Lockscreen */
+
+const {GObject, Gtk} = imports.gi;
+
+const _css = `
+.lockscreen {
+    background: url('/usr/share/icons/Adwaita/256x256/status/dialog-password.png'),
```

Comment:
I know this is a stub, but would `-gtk-icontheme` work instead of hardcoding a path?

Reply:
I don't think so, as we don't call `gtk_render_icon()` anywhere in this widget.